modin/conftest.py

```diff
@@ -567,76 +573,73 @@ def s3_base(worker_id):
         URL for motoserver/moto CI service.
     """
     # copied from pandas conftest.py
-    with pandas._testing.ensure_safe_environment_variables():
-        # still need access keys for https://github.com/getmoto/moto/issues/1924
-        os.environ.setdefault("AWS_ACCESS_KEY_ID", CIAWSAccessKeyID.get())
-        os.environ.setdefault("AWS_SECRET_ACCESS_KEY", CIAWSSecretAccessKey.get())
-        os.environ["AWS_REGION"] = "us-west-2"
-        if GithubCI.get():
-            if sys.platform in ("darwin", "win32", "cygwin") or (
-                platform.machine() in ("arm64", "aarch64")
-                or platform.machine().startswith("armv")
-            ):
-                # pandas comments say:
-                # DO NOT RUN on Windows/macOS/ARM, only Ubuntu
-                # - subprocess in CI can cause timeouts
-                # - GitHub Actions do not support
-                #   container services for the above OSs
-                pytest.skip(
-                    (
-                        "S3 tests do not have a corresponding service in Windows, macOS "
-                        + "or ARM platforms"
-                    )
+    # still need access keys for https://github.com/getmoto/moto/issues/1924
+    monkeysession.setenv("AWS_ACCESS_KEY_ID", "foobar_key")
+    monkeysession.setenv("AWS_SECRET_ACCESS_KEY", "foobar_secret")
+    monkeysession.setenv("AWS_REGION", "us-west-2")
+    if GithubCI.get():
+        if sys.platform in ("darwin", "win32", "cygwin") or (
+            platform.machine() in ("arm64", "aarch64")
+            or platform.machine().startswith("armv")
+        ):
+            # pandas comments say:
+            # DO NOT RUN on Windows/macOS/ARM, only Ubuntu
+            # - subprocess in CI can cause timeouts
+            # - GitHub Actions do not support
+            #   container services for the above OSs
+            pytest.skip(
+                (
+                    "S3 tests do not have a corresponding service in Windows, macOS "
+                    + "or ARM platforms"
                 )
```

Comment:
do we need two sets of braces? can we remove one set of them?

Reply:
Removed